cylc/uiserver/app.py

```diff
@@ -546,10 +547,17 @@ def set_auth(self) -> Authorization:
         """Create authorization object.
         One for the lifetime of the UIServer.
         """
+        user_auth = self.config.CylcUIServer.user_authorization
+        site_auth = self.config.CylcUIServer.site_authorization
+        if isinstance(user_auth, LazyConfigValue):
+            user_auth: dict = user_auth.to_dict()
```

Comment:
Need to put the type annotation on the initial definition for Mypy to be happy `Union[LazyConfigValue, dict]`